probes/testsRunInCI/def.yml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: testsRunInCI
+short: Checks that the project runs tests in the CI for example with Github Actions or Prow.
+motivation: >
+  Running tests helps developers catch mistakes early on, which can reduce the number of vulnerabilities that find their way into a project.
+implementation: >
+ The probe checks for tests in the projects CI jobs in the recent commits (~30).
+outcome:
+  - The probe returns one OutcomePositive for each successful test or check that ran in the projects CI. All findings include the number of merged pull requests as well as how many of these pull requests were tested by tests in the CI. These fields are called "totalMerged" and "totalTested".
```

Comment:
this will need to change to reflect removal of values

Reply:
Done in https://github.com/ossf/scorecard/pull/3621/commits/a7970c6231277f2d808da2716c3e28f8a85b6948